transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -293,6 +296,13 @@ public void showCurrencySelectionDialog(@NonNull final TreeMap<String, String> c
     @Override
     public void showAddTransferMethod(@NonNull final String country, @NonNull final String currency,
             @NonNull final String transferMethodType, @NonNull final String profileType) {
+        Map<String, String> params = new HashMap<>();
+        params.put("hyperwallet_ea_country", country);
+        params.put("hyperwallet_ea_currency", currency);
+        params.put("hyperwallet_ea_type", transferMethodType);
```

Comment:
when core insight is ready please use the following constants:
`InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_COUNTRY`
`InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_CURRENCY`
`InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_TYPE`